rest-service/manager_rest/test/endpoints/test_plugins_updates.py

```diff
@@ -154,6 +153,30 @@ def test_raises_while_plugins_updates_are_active(self):
                 '{0}'.format(plugins_update.id)):
             self.client.plugins_update.update_plugins('hello_world')
 
+    def test_doesnt_raise_when_plugins_updates_are_not_active(self):
+        self.put_file(*self.put_blueprint_args(blueprint_id='hello_world'))
+        self.client.deployments.create('hello_world', 'd123')
+        self.wait_for_deployment_creation(self.client, 'd123')
+        plugins_update_id = self.client.plugins_update.update_plugins(
+            'hello_world').id
+        plugins_update = self._sm.get(models.PluginsUpdate, plugins_update_id)
+        plugins_update.state = STATES.SUCCESSFUL
+        self._sm.update(plugins_update)
+
+        plugins_update_id = self.client.plugins_update.update_plugins(
+            'hello_world').id
+        plugins_update = self._sm.get(models.PluginsUpdate, plugins_update_id)
+        plugins_update.state = STATES.FAILED
+        self._sm.update(plugins_update)
+
+        plugins_update_id = self.client.plugins_update.update_plugins(
```

Comment:
may does this make sense? would not it "fix" the update?

Reply:
Why it makes sense* ?

This just checks that an update can be run after a "non-active" plugin update has been made, which is similar to what you thought _should_ happen.

Reply:
Oh, so you need to split this test to the different scenerios you are testing for.
It's not clear

Reply:
I split the tests